### ISSUE TITLE
syntax error, unexpected ':', expecting keyword_end (SyntaxError)

### ISSUE THREAD

**Issue by ketandoshi** [railslove/rack-tracker#154]
When started the server after installing the rack-tracker version 1.12.1,
it raises an error as below:

```
/.rvm/gems/ruby-2.2.10/gems/rack-tracker-1.12.1/lib/rack/tracker/google_global/google_global.rb:62: syntax error, unexpected ':', expecting keyword_end (SyntaxError)
      WARNING: One of the trackers specifie...
              ^
/.rvm/gems/ruby-2.2.10/gems/rack-tracker-1.12.1/lib/rack/tracker/google_global/google_global.rb:62: syntax error, unexpected tIDENTIFIER, expecting :: or '[' or '.'
...ified for Rack::Tracker handler 'google_global' is empty.
```

**Comment by bumi** [comment]
what ruby version do you use? 
but the main problem seems to be that your google_global tracker configuration is wrong. Can you review that? Do you have an id configured there?

**Comment by ketandoshi** [comment]
ruby version 2.2.10

google analytics is configured as below:

```
config.middleware.use(Rack::Tracker) do
  handler :google_analytics, { tracker: 'ga-key', ecommerce: true }
end
```

**Comment by DonSchado** [comment]
2.2. was EOL 2 years ago, are you able to use a newer Ruby, or is this a legacy environment?

**Comment by ketandoshi** [comment]
@DonSchado 
Actually mine is a legacy environment. I can give it a try for newer version to check.
But if people are using legacy environment, then it would be a problem for sure I guess.

**Comment by DonSchado** [comment]
Reading the code in [google_global.rb](https://github.com/railslove/rack-tracker/blob/master/lib/rack/tracker/google_global/google_global.rb#L61) the only obvious non 2.2. Syntax is the "squiggly" HEREDOC `<<~` 

If you make a PR to fix that and test if the rest works with 2.2., I'm happy to make a new release for you.

**Comment by bumi** [comment]
@ketandoshi any update on this? could you get it working? (if so, a PR would be nice ;)
